src/adapters/network.js

```diff
@@ -72,13 +86,17 @@ const httpSend = async (options) => {
  *
  * handles http GET requests returns promise as a response throws error in case of non 2XX statuses
  */
-const httpGET = async (url, options) => {
+const httpGET = async (url, options, statTags = {}) => {
```

Comment:
can use httpSend method for all httpGET, httpPOST etc? @sanpj2292

Reply:
Yes. But that's different to the intention of this PR. We can probably take it up separately